docs/reference/adobe/experience/decisioning/scope-details.schema.md

```diff
@@ -81,6 +82,30 @@ The activity that was a requested or chosen for this decision scope. Some scopes
 
 
 
+## xdm:characteristics
+### Scope Details Characteristics
+
+Additional properties or attributes belonging to this particular scope details.
```

Comment:
How about - `Additional properties or attributes belonging to this particular decision scope.`

Reply:
Correct. The key's do not need to be namespaced unless you want it that way.

Reply:
thank you